packages/storybook/.storybook/preview.tsx

```diff
@@ -50,6 +50,7 @@ export const parameters = {
           'React Hooks',
           [
             'Setting up',
+            'Adapters',
```

Comment:
Adapters aren't really react hooks, is there a way to have this in the composite section without it being weird?

Reply:
Hmm the do interface with state so I see how its useful under statefulclient section